### PR TITLE
chore(DTFS2-7423): refactor unissued-facilities/MIN tests

### PR DESCRIPTION
**Pull request by MarRobSoftwire**
## Introduction :pencil2:
We want to refactor how ffs are tested to remove the if statements from tests.

## Resolution :heavy_check_mark:
Move unissued-facilities/MIA tests into facility-end-date-enabled & facility-end-date-disabled directories

### CODE REVIEW

File: e2e-tests/gef/cypress/e2e/facility/unissued-facilities/MIN/facility-end-date-disabled/unissued-facilities-change-to-issued-3-months-after-MIN.spec.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+let dealId;
+let token;
+let facilityOneId;
+
+const { unissuedCashFacility, issuedCashFacility, unissuedContingentFacility, unissuedCashFacilityWith20MonthsOfCover } = multipleMockGefFacilities();
+
+const FACILITY_THREE_SPECIAL = { ...unissuedContingentFacility };
+FACILITY_THREE_SPECIAL.specialIssuePermission = true;
+
+context('Unissued Facilities MIN - change to issued more than 3 months after MIN submission date', () => {
```
Comment (ttbarnes) [inline e2e-tests/gef/cypress/e2e/facility/unissued-facilities/MIN/facility-end-date-disabled/unissued-facilities-change-to-issued-3-months-after-MIN.spec.js:25]: Can we add a FF description (disabled/enabled) for all the appropriate tests?

**Review by ttbarnes — CHANGES_REQUESTED** [review]

File: e2e-tests/gef/cypress/e2e/facility/unissued-facilities/MIN/about-unissued-facility.spec.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
+import { format } from 'date-fns';
+
+import relative from '../../../relativeURL';
+
+import CONSTANTS from '../../../../fixtures/constants';
+
+import dateConstants from '../../../../../../e2e-fixtures/dateConstants';
+
```
Comment (ttbarnes) [inline e2e-tests/gef/cypress/e2e/facility/unissued-facilities/MIN/about-unissued-facility.spec.js:8]: Can we remove these empty lines inbetween imports for E2E tests in this PR please

**Review by ttbarnes — CHANGES_REQUESTED** [review]
One minor comment RE empty line imports

Not for this PR because this is only moving the tests for FF enabled/disabled - would like to further refactor these tests because:

- There is a lot of repetition
- Some tests are very long, some have 600+ lines of tests

File: e2e-tests/gef/cypress/e2e/facility/unissued-facilities/MIN/about-unissued-facility.spec.js (hunk 2 above)
Comment (ttbarnes) [reply e2e-tests/gef/cypress/e2e/facility/unissued-facilities/MIN/about-unissued-facility.spec.js:8]: Actually, going to resolve this as i've made the same changes in [this PR](https://github.com/UK-Export-Finance/dtfs2/pull/3556)

Will address in there

**Review by ttbarnes — APPROVED** [review]

File: e2e-tests/gef/cypress/e2e/facility/unissued-facilities/MIN/about-unissued-facility.spec.js [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    it('displays error messages when cover end date is same as the cover start date', () => {
+      applicationPreview.unissuedFacilitiesReviewLink().click();
+      unissuedFacilityTable.updateIndividualFacilityButton(0).click();
+
+      cy.keyboardInput(aboutFacilityUnissued.coverStartDateDay().clear(), dateConstants.todayDay);
+      cy.keyboardInput(aboutFacilityUnissued.coverStartDateMonth().clear(), dateConstants.todayMonth);
+      cy.keyboardInput(aboutFacilityUnissued.coverStartDateYear().clear(), dateConstants.todayYear);
+      cy.keyboardInput(aboutFacilityUnissued.coverEndDateDay().clear(), dateConstants.todayDay);
+      cy.keyboardInput(aboutFacilityUnissued.coverEndDateMonth().clear(), dateConstants.todayMonth);
+      cy.keyboardInput(aboutFacilityUnissued.coverEndDateYear().clear(), dateConstants.todayYear);
```
Comment (ttbarnes) [inline e2e-tests/gef/cypress/e2e/facility/unissued-facilities/MIN/about-unissued-facility.spec.js:226]: Need to remove `.clear()` from all instances of `cy.keyboardInput` in this test (currently failing), e.g should be: 

```js
cy.keyboardInput(aboutFacilityUnissued.coverEndDateYear(), dateConstants.todayYear);
```

Comment (BethThomas141) [reply e2e-tests/gef/cypress/e2e/facility/unissued-facilities/MIN/about-unissued-facility.spec.js]: Good spot, have done 👍

**Review by ttbarnes — DISMISSED** [review]

**Pull request merged**